### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by ChristinaLK**
Updates to: 
- readme
- lesson scope / structure
- outline of episodes

I still need to fix the README links, otherwise let me know what you think.

### CODE REVIEW

**Comment by dbrunson** [comment]
Wow, this looks great @ChristinaLK!

File: _episodes/00-intro.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
-title: "Introducing high-performance computing"
+title: "Introducing High-Performance Computing"
 questions:
-- Why use a high-performance computing cluster?
+- Why use a large-scale computing system?
 ---
+
+Topics
+
+When it is appropriate to use a cluster
```
Comment (shwina) [inline _episodes/00-intro.md:12]: I would also add:

How is using a large-scale computing system different from using your laptop? Talk to your neighbor about some differences you may already know about, and some differences/difficulties you imagine you may run into.

Comment (shwina) [reply _episodes/00-intro.md:12]: I think it's important that learners start thinking early on in the workshop about this. I also feel it's important to do away quickly with any misconceptions, fears or unreasonable expectations they may have about HPC.

File: _episodes/02-shell-basics.md [hunk 2]
```diff
@@ -1,8 +1,20 @@
 ---
-title: "Shell basics"
+title: "Introduction to the Shell"
 questions:
 - "How to look at files and folders using the shell?"
 - "How are files and folders organized on this computer?"
 - "How to create folders and files?"
 - "How to specify the location of a file or folder?"
+- "What is a shell script and how do I write one?"
```
Comment (shwina) [inline _episodes/02-shell-basics.md]: Going from learning basic commands to navigate and manipulate the file system (`ls`, `pwd`, `cd`, `rm`, `mv`, `cp`) to writing shell scripts is a **big** leap for learners. Also, they may not see value in a script that just  uses these commands.

I think that some core ideas need to be covered first:

1. The shell is more than a way to look at files or folders. It's also a way to run programs.
1. Programs may expect data as input and may produce data as output. The shell lets us control which programs to run on which inputs and what to do with the output.

File: _episodes/04-scheduler.md [hunk 3]
```diff
@@ -4,3 +4,26 @@ questions:
 - "How many and what kind of compute nodes are on this cluster?"
 - "How does the cluster get my tasks or programs to the compute nodes?"
 ---
+
+Topics: 
+
+Description of compute notes
```
Comment (shwina) [inline _episodes/04-scheduler.md]: notes->nodes

File: target-audience.md [hunk 4]
```diff
@@ -12,3 +12,24 @@ simply not be run on smaller computers
 
 and aims to cover the essential skills required to be able to get
 started with using such resources.
+
+This lesson is intended for **novices**.  The expectation is that participants 
+have *maybe* used a command line interface, and possibly programmed 
+something python or R, but have not used the command line extensively,
```
Comment (shwina) [inline target-audience.md]: python->Python

File: _extras/guide.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+Because different compute resources have different features, this 
+lesson, by necessity, should be thought of as completely modular.  
+We have here the basic topics that might be required, in a recommended order, 
+but additions and deletions are encouraged, based on the needs of the 
+audience and the compute resource that is being used.  It could also be 
+expanded or contracted to fill a certain amount of time -- the default 
+time period should be about day, but for a complete novice audience 
+(has **never** used the command line), it should be more like a day 
+and a half, or two days.
```
Comment (shwina) [inline _extras/guide.md]: I like this concept!

File: _episodes/04-scheduler.md [hunk 6]
```diff
@@ -4,3 +4,26 @@ questions:
...
+> * Have one third of the room be "users", given several slips of 
+> paper (or post-it notes) and pens
+> * Have two thirds of the room be "compute nodes"
+> 
+> "Users" should fill out their slips of paper with math problems.  
+> These math problems represent any kind of computation that you 
+> might want to run on the cluster.  How will these computations 
+> get to the compute nodes?  The "scheduler" will need to collect 
+> the math problems from the "users" and deliver them to the "compute 
+> nodes", returning the results when the node is done.
```
Comment (shwina) [inline _episodes/04-scheduler.md]: I tried this and we ended up using TONS of sticky notes; and also it mostly left everyone confused about what really was going on. I love this idea, but some more specific instructions to the instructor would be great :-)

**Review by shwina — COMMENTED** [review]
Thank you so much for this, @ChristinaLK!

File: _episodes/04-scheduler.md (hunk 6 above)
Comment (ChristinaLK) [reply _episodes/04-scheduler.md]: I'll add something to the instructor's guide.

**Comment by shwina** [comment]
LGTM. @psteinb, thoughts?

**Comment by ChristinaLK** [comment]
@shwina @psteinb , I can't merge in this repo anymore -- can you confirm that you can?

**Comment by jduckles** [comment]
@ChristinaLK fixed your merge permissions.

**Pull request merged**